repose-aggregator/docs/src/asciibinder/welcome/release-notes.adoc

```diff
@@ -3,7 +3,20 @@
 == 9.0.0.0 (2018-??-??)
 <<ver-9-upgrade-notes.adoc, Upgrade Notes>>
 
-* https://repose.atlassian.net/browse/REP-4992[REP-4922] - Removed ambiguous setting for chunked encoding options on <<services/http-connection-pool.adoc, HTTP Client/Connection Pool Service>>.
+* https://repose.atlassian.net/browse/REP-4992[REP-4922] - Removed ambiguous setting for chunked encoding options on <<services/http-client.adoc, HTTP Client/Connection Pool Service>>.
+* https://repose.atlassian.net/browse/REP-3589[REP-3589] - Rewrote the <<services/http-client.adoc#, HTTP Client Service>>.
```

Comment:
A lot of these bullets seem like they should be in the upgrade notes not here.

Reply:
I was split on that. I ended up putting them here following the idea a change modifies the code running on a user's machine, and therefore the change should be documented in the release notes. Then I put more details on the upgrade page.